lib/utils/github.ts

```diff
@@ -16,23 +16,63 @@ export async function addComment({
 }): Promise<string> {
 	const commentResult = await octokit.graphql(
 		`
-    mutation($issueId: ID!, $comment: String!) {
-      addComment(input: { subjectId: $issueId, body: $comment }) {
-        clientMutationId
-      }
-    }
+			mutation($issueId: ID!, $comment: String!) {
+				addComment(input: { subjectId: $issueId, body: $comment }) {
+					clientMutationId
+				}
+			}
     `,
-		{
-			issueId,
-			comment
-		}
+		{issueId, comment}
 	)
 
 	if (!commentResult) throw new Error('Could not add comment')
 
 	return commentResult
 }
 
+export async function getRepoMeta({
+	name,
+	owner,
+	octokit
+}: {
+	name: string
+	owner: string
+	octokit: any
+}): Promise<{
+	labels: Label[]
+	description: string
+}> {
+	const res = await octokit.graphql(
+		`
+			query Labels($name: String!, $owner: String!) {
```

Comment:
what are your thoughts of using octokit.request instead of graphql. Makes it easier I guess to find issues since there's type checking and stuff as well as more readability. More limited than graphql in some extreme cases, but really useful generally.

Reply:
Good point! In this case it's nice since we can grab repo description + labels in one request, but the others can maybe be replaced by `.request()`. Will try.